diff --git a/app/Actions/Photo/Extensions/VideoEditing.php b/app/Actions/Photo/Extensions/VideoEditing.php
index 1a96769..5842606 100644
--- a/app/Actions/Photo/Extensions/VideoEditing.php
+++ b/app/Actions/Photo/Extensions/VideoEditing.php
@@ -45,7 +45,10 @@ trait VideoEditing
                /**
                 * ! check if we can use path instead of this ugly thing.
                 */
-               $ffmpeg = FFMpeg::create();
+               $ffmpeg = FFMpeg::create(array(
+                       'ffmpeg.binaries'  => '/usr/bin/ffmpeg',
+                       'ffprobe.binaries' => '/usr/bin/ffprobe',
+               ));
                /** @var Video */
                $video = $ffmpeg->open(Storage::path('big/' . $photo->url));
                if (!($tmp = tempnam(sys_get_temp_dir(), 'lychee')) ||
