diff --git a/vendor/lychee-org/php-exif/lib/PHPExif/Adapter/FFprobe.php b/vendor/lychee-org/php-exif/lib/PHPExif/Adapter/FFprobe.php
index 9ececfe..106eadd 100644
--- a/vendor/lychee-org/php-exif/lib/PHPExif/Adapter/FFprobe.php
+++ b/vendor/lychee-org/php-exif/lib/PHPExif/Adapter/FFprobe.php
@@ -40,7 +40,7 @@ class FFprobe extends AdapterAbstract
      *
      * @var string
      */
-    protected string $toolPath = '';
+    protected string $toolPath = '/usr/bin/ffprobe';

     /**
      * @var string-e
