diff --git a/app/Actions/Photo/Extensions/VideoEditing.php b/app/Actions/Photo/Extensions/VideoEditing.php
index 1a96769..5842606 100644
--- a/app/Actions/Photo/Extensions/VideoEditing.php
+++ b/app/Actions/Photo/Extensions/VideoEditing.php
@@ -45,7 +45,10 @@ trait VideoEditing
                /**
                 * ! check if we can use path instead of this ugly thing.
                 */
-               $ffmpeg = FFMpeg::create();
+               $ffmpeg = FFMpeg::create(array(
+                       'ffmpeg.binaries'  => '/usr/bin/ffmpeg',
+                       'ffprobe.binaries' => '/usr/bin/ffprobe',
+               ));
                /** @var Video */
                $video = $ffmpeg->open(Storage::path('big/' . $photo->url));
                if (!($tmp = tempnam(sys_get_temp_dir(), 'lychee')) ||
diff --git a/vendor/lychee-org/php-exif/lib/PHPExif/Adapter/FFprobe.php b/vendor/lychee-org/php-exif/lib/PHPExif/Adapter/FFprobe.php
index 9ececfe..106eadd 100644
--- a/vendor/lychee-org/php-exif/lib/PHPExif/Adapter/FFprobe.php
+++ b/vendor/lychee-org/php-exif/lib/PHPExif/Adapter/FFprobe.php
@@ -33,7 +33,7 @@ class FFprobe extends AdapterAbstract
      *
      * @var string
      */
-    protected $toolPath;
+    protected $toolPath = '/usr/bin/ffprobe';

     /**
      * @var string
      